joboard/settings.py

```diff
@@ -124,3 +124,10 @@
 STATIC_URL = '/static/'
 LOGIN_REDIRECT_URL = '/'
 LOGOUT_REDIRECT_URL = '/'
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_USE_TLS = True
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'JoBoardT@gmail.com'
+EMAIL_HOST_PASSWORD = 'Jb123456'
```

Comment:
If it was a real application you can't write the password like that (visible for everyone).... 
Better save it in the DB

Reply:
deleted this for now. I will upload that in a separate PR